pkg/version/version.go

```diff
@@ -80,9 +80,19 @@ func splitVersion(version string) (string, string) {
 	// Version from an automated container build environment(not a tag) or a local build. For example v20201009-v0.18.0-46-g939c1c0.
 	m2, _ := regexp.MatchString(`^v\d{8}-v\d+\.\d+\.\d+-\w+-\w+$`, version)
 
+	// Version tagged by helm chart releaser action
+	helm, _ := regexp.MatchString(`^v\d{8}-descheduler-helm-chart-\d+\.\d+\.\d+$`, version)
+	// Dirty version where helm chart is the last known tag
+	helm2, _ := regexp.MatchString(`^v\d{8}-descheduler-helm-chart-\d+\.\d+\.\d+-\w+-\w+$`, version)
```

Comment:
can probably be moved closer to where the variables used.

Reply:
true, but I also like having all the regex's near each other

Reply:
fair enough